### PR TITLE
[Doc]Update table schema for version 2.4+ audit loader

### PR DESCRIPTION
**Pull request by DanJing-coder**
Update table schema for version 2.4+ audit loader

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/administration/audit_loader.md [hunk 1]
```diff
@@ -121,7 +126,7 @@ CREATE TABLE starrocks_audit_db__.starrocks_audit_tbl__
     digest           VARCHAR(32)            COMMENT "SQL fingerprint"
 ) engine=OLAP
 duplicate key(query_id, time, client_ip)
-partition by range(time) ()
+partition by range(time) () BUCKETS 3
```
Comment (evelynzhaojie) [inline docs/administration/audit_loader.md:129]: buckets is misplaced, it should follow distributed by

File: docs/administration/audit_loader.md [hunk 2]
```diff
@@ -156,7 +161,7 @@ CREATE TABLE starrocks_audit_db__.starrocks_audit_tbl__
     digest          VARCHAR(32)           COMMENT "SQL fingerprint"
 ) engine=OLAP
 DUPLICATE KEY(query_id, time, client_ip)
-PARTITION BY RANGE(time) ()
+PARTITION BY RANGE(time) () BUCKETS 3
```
Comment (evelynzhaojie) [inline docs/administration/audit_loader.md:164]: same

File: docs/administration/audit_loader.md [hunk 3]
```diff
@@ -190,7 +195,7 @@ CREATE TABLE starrocks_audit_db__.starrocks_audit_tbl__
     stmt            STRING                COMMENT "SQL statement"
 ) engine=OLAP
 DUPLICATE KEY(query_id, time, client_ip)
-PARTITION BY RANGE(time) ()
+PARTITION BY RANGE(time) () BUCKETS 3
```
Comment (evelynzhaojie) [inline docs/administration/audit_loader.md:198]: same

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.2

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Pull request merged**